src/resolvers/resolver.ts

```diff
@@ -51,6 +79,15 @@ const resolvers = {
       return user;
     },
   },
+  // Role: {
+  //   async role(parent: any) {
+  //     const role = await Role.findOne({
+  //       where: { _id: parent.user._id.toString() },
```

Comment:
This will be needed. Instead of searching for the id though we would check for the `name` as it will be unique in the enum

Reply:
Solved.